### PR TITLE
Added method to get the current read position to sf::Packet.

### PR DESCRIPTION
**Pull request by 1aam2am1**
https://en.sfml-dev.org/forums/index.php?topic=19880.msg144308#msg144308

### CODE REVIEW

**Comment by LaurentGomila** [comment]
I'd rather name it `getReadPosition`. And it should be `const`.

Also, the documentation is really useless as it is, I'm sure we can find something more relevant 😛

**Comment by 1aam2am1** [comment]
Yes, someone please write a good documentation and I will amend this.
This request was something from my past. And I would like to return to sfml and has this function.

File: include/SFML/Network/Packet.hpp [hunk 1]
```diff
@@ -72,10 +72,21 @@ class SFML_NETWORK_API Packet
...
     ///
     ////////////////////////////////////////////////////////////
     void append(const void* data, std::size_t sizeInBytes);
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get a current reading position
+    ///
+    /// \return Current position
+    ///
+    /// \see append
```
Comment (eXpl0it3r) [inline include/SFML/Network/Packet.hpp]: Requires better documentation

File: include/SFML/Network/Packet.hpp [hunk 2]
```diff
@@ -72,10 +72,21 @@ class SFML_NETWORK_API Packet
     /// \param sizeInBytes Number of bytes to append
     ///
     /// \see clear
+    /// \see readCurrentReadPosition
```
Comment (binary1248) [inline include/SFML/Network/Packet.hpp]: This line needs to be updated.

**Comment by 1aam2am1** [comment]
Sorry for the lack of responsiveness but I should probably say that I currently (and for the foreseeable future) can't invest time in SFML. Please feel free to take over this PR.
You should write your own code with this functionality and pull it with future version of sfml

**Comment by eXpl0it3r** [comment]
Anyone willing to update this?

**Comment by eXpl0it3r** [comment]
Superseded by #1641

**Pull request closed without merging**